src/sensors/SensorManager.h

```diff
@@ -37,26 +37,37 @@ namespace SlimeVR
         {
         public:
             SensorManager()
-                : m_Logger(SlimeVR::Logging::Logger("SensorManager")), m_Sensor1(new EmptySensor(0)), m_Sensor2(new EmptySensor(0)) {}
+                : m_Logger(SlimeVR::Logging::Logger("SensorManager")) {
+                    for (auto & u : m_Sensors) {
+                        u = new EmptySensor(0);
+                    }
+                }
             ~SensorManager()
             {
-                delete m_Sensor1;
-                delete m_Sensor2;
+                for (auto u : m_Sensors) {
+                    if (u != NULL) {
+                        delete u;
+                    }
+                }
             }
 
             void setup();
             void postSetup();
 
             void update();
-
-            Sensor *getFirst() { return m_Sensor1; };
-            Sensor *getSecond() { return m_Sensor2; };
+            
+            Sensor  ** const getSensors() { return m_Sensors; };
```

Comment:
Not sure if this should be a std::vector or something, I feel like this could be worrysome with memory safety. (I don't know C++ that well though)

Reply:
I'm not sure if it's good to use dynamic allocated memory here, but that should do the job (and fix many other weird-looking loops)

Reply:
```suggestion
            Sensor ** const getSensors() { return m_Sensors; };
```
oh yeah, there was a double space in here

Reply:
That array is replaced with vector in the latest commit and thus the double space is gone -w-/